### PR TITLE
Added Start Quiz button to dashboard page

### PR DESCRIPTION
**Pull request by mounika-puppala**

### CODE REVIEW

**Review by ahujasushant — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,18 @@
 import React, { Component } from 'react';
 import './App.css';
+import { BrowserRouter as Router, Route, Link} from "react-router-dom";
```
Comment (manu29d) [inline src/App.js]: Unused imported variable `Link`

File: src/dashboard/instructions.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
```
Comment (manu29d) [inline src/dashboard/instructions.js]: rename file with upper case first letter `Instructions`

File: src/dashboard/startQuiz.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
```
Comment (manu29d) [inline src/dashboard/startQuiz.js]: Rename file with upper case first letter and a descriptive name`StartQuizButton`

**Review by manu29d — CHANGES_REQUESTED** [review]

File: src/dashboard/startQuiz.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+import { Link } from 'react-router-dom';
```
Comment (sushantahujaa) [inline src/dashboard/startQuiz.js]: I already did the startquiz button component part. I think but it's cool we can merge this in.

**Review by manu29d — APPROVED** [review]

**Pull request merged**